src/main/proto/ga4gh/bio_metadata.proto

```diff
@@ -155,19 +161,6 @@ message BioCharacteristic {
   //      termId: "HP:0001714"
   //      label: "Ventricular hypertrophy"
   //
-  repeated OntologyTerm negatedOntologyTerms = 3;
-
-}
-
-// Characteristics is a wrapper object for the diseases, phenotypes associated
-// with a first class BioMetadata object (Individual, BioSample).
-message Characteristics {
-  // Diseases associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic diseases = 1;
-
-  // Phenotypes associated with the parent BioMetadata object (Individual,
-  // BioSample).
-  repeated BioCharacteristic phenotypes = 2;
+  repeated OntologyTerm negatedOntologyTerms = 4;
 
```

Comment:
this looks like a simple re-organization of existing fields, so we can keep the original protobuf number of 3